Enable benchmarks on Linux/ARM

I'm trying to run benchmarks on Linux/ARM target but faced with some issues.
First is can be fixed with #516 PR.
Another one it's look to be related to SDK (or may be something incorrect with my environment):
```
python3 scripts/benchmarks_ci.py -f netcoreapp3.0 --architecture arm --filter '*'
[2019/05/29 16:54:32][INFO] ----------------------------------------------
[2019/05/29 16:54:32][INFO] Initializing logger 2019-05-29 16:54:32.483575
[2019/05/29 16:54:32][INFO] ----------------------------------------------
[2019/05/29 16:54:32][INFO] Installing tools.
[2019/05/29 16:54:32][INFO] ----------------------
[2019/05/29 16:54:32][INFO] Downloading DotNet Cli
[2019/05/29 16:54:32][INFO] ----------------------
[2019/05/29 16:54:32][INFO] DotNet Install Path: '/home/asoldatov/performance/tools/dotnet/arm'
[2019/05/29 16:54:32][INFO] Downloading https://dot.net/v1/dotnet-install.sh
[2019/05/29 16:54:34][INFO] $ pushd "/home/asoldatov/performance"
[2019/05/29 16:54:34][INFO] $ /home/asoldatov/performance/tools/dotnet/arm/dotnet-install.sh -InstallDir /home/asoldatov/performance/tools/dotnet/arm -Architecture arm -Channel master
[2019/05/29 16:54:36][INFO] dotnet-install: .NET Core SDK version 3.0.100-preview6-012105 is already installed.
[2019/05/29 16:54:36][INFO] dotnet-install: Adding to current process PATH: `/home/asoldatov/performance/tools/dotnet/arm`. Note: This change will be visible only when sourcing script.
[2019/05/29 16:54:36][INFO] dotnet-install: Installation finished successfully.
[2019/05/29 16:54:36][INFO] $ popd
[2019/05/29 16:54:36][INFO] -----------------------------
[2019/05/29 16:54:36][INFO] Downloading BenchView scripts
[2019/05/29 16:54:36][INFO] -----------------------------
[2019/05/29 16:54:38][INFO] http://benchviewtestfeed.azurewebsites.net/api/v2/package/microsoft.benchview.jsonformat/0.1.0-pre030 -> /home/asoldatov/performance/tools/Microsoft.BenchView.JSONFormat
[2019/05/29 16:54:45][INFO] $ dotnet --info
[2019/05/29 16:54:46][INFO] .NET Core SDK (reflecting any global.json):
[2019/05/29 16:54:46][INFO]  Version:   3.0.100-preview6-012105
[2019/05/29 16:54:46][INFO]  Commit:    256eaa5e2f
[2019/05/29 16:54:46][INFO] 
[2019/05/29 16:54:46][INFO] Runtime Environment:
[2019/05/29 16:54:46][INFO]  OS Name:     ubuntu
[2019/05/29 16:54:46][INFO]  OS Version:  16.04
[2019/05/29 16:54:46][INFO]  OS Platform: Linux
[2019/05/29 16:54:46][INFO]  RID:         ubuntu.16.04-arm
[2019/05/29 16:54:46][INFO]  Base Path:   /home/asoldatov/performance/tools/dotnet/arm/sdk/3.0.100-preview6-012105/
[2019/05/29 16:54:46][INFO] 
[2019/05/29 16:54:46][INFO] Host (useful for support):
[2019/05/29 16:54:46][INFO]   Version: 3.0.0-preview6-27728-04
[2019/05/29 16:54:46][INFO]   Commit:  b2f8fc281d
[2019/05/29 16:54:46][INFO] 
[2019/05/29 16:54:46][INFO] .NET Core SDKs installed:
[2019/05/29 16:54:46][INFO]   3.0.100-preview6-012105 [/home/asoldatov/performance/tools/dotnet/arm/sdk]
[2019/05/29 16:54:46][INFO] 
[2019/05/29 16:54:46][INFO] .NET Core runtimes installed:
[2019/05/29 16:54:46][INFO]   Microsoft.AspNetCore.App 3.0.0-preview6-19278-05 [/home/asoldatov/performance/tools/dotnet/arm/shared/Microsoft.AspNetCore.App]
[2019/05/29 16:54:46][INFO]   Microsoft.NETCore.App 3.0.0-preview6-27728-04 [/home/asoldatov/performance/tools/dotnet/arm/shared/Microsoft.NETCore.App]
[2019/05/29 16:54:46][INFO] 
[2019/05/29 16:54:46][INFO] To install additional .NET Core runtimes or SDKs:
[2019/05/29 16:54:46][INFO]   https://aka.ms/dotnet-download
[2019/05/29 16:54:46][INFO] -------------------------------
[2019/05/29 16:54:46][INFO] Restoring .NET micro benchmarks
[2019/05/29 16:54:46][INFO] -------------------------------
[2019/05/29 16:54:46][INFO] $ pushd "/home/asoldatov/performance/src/benchmarks/micro"
[2019/05/29 16:54:46][INFO] $ dotnet restore /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --packages /home/asoldatov/performance/artifacts/packages
[2019/05/29 16:54:51][INFO]   Restore completed in 55.19 ms for /home/asoldatov/performance/src/tools/Reporting/Reporting/Reporting.csproj.
[2019/05/29 16:54:51][INFO]   Restore completed in 45.72 ms for /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj.
[2019/05/29 16:54:51][INFO]   Restore completed in 55.92 ms for /home/asoldatov/performance/src/harness/BenchmarkDotNet.Extensions/BenchmarkDotNet.Extensions.csproj.
[2019/05/29 16:54:51][INFO] $ popd
[2019/05/29 16:54:51][INFO] --------------------------------------------------
[2019/05/29 16:54:51][INFO] Building .NET micro benchmarks for 'netcoreapp3.0'
[2019/05/29 16:54:51][INFO] --------------------------------------------------
[2019/05/29 16:54:51][INFO] $ pushd "/home/asoldatov/performance/src/benchmarks/micro"
[2019/05/29 16:54:51][INFO] $ dotnet build /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --configuration Release --framework netcoreapp3.0 --no-restore
[2019/05/29 16:54:52][INFO] Microsoft (R) Build Engine version 16.2.0-preview-19274-03+103f944e0 for .NET Core
[2019/05/29 16:54:52][INFO] Copyright (C) Microsoft Corporation. All rights reserved.
[2019/05/29 16:54:52][INFO] 
[2019/05/29 16:55:02][INFO]   Reporting -> /home/asoldatov/performance/artifacts/bin/Reporting/Release/netcoreapp3.0/Reporting.dll
[2019/05/29 16:55:07][INFO]   BenchmarkDotNet.Extensions -> /home/asoldatov/performance/artifacts/bin/BenchmarkDotNet.Extensions/Release/netcoreapp3.0/BenchmarkDotNet.Extensions.dll
[2019/05/29 16:55:24][INFO]   MicroBenchmarks -> /home/asoldatov/performance/artifacts/bin/MicroBenchmarks/Release/netcoreapp3.0/MicroBenchmarks.dll
[2019/05/29 16:55:24][INFO] 
[2019/05/29 16:55:24][INFO] Build succeeded.
[2019/05/29 16:55:24][INFO]     0 Warning(s)
[2019/05/29 16:55:24][INFO]     0 Error(s)
[2019/05/29 16:55:24][INFO] 
[2019/05/29 16:55:24][INFO] Time Elapsed 00:00:31.74
[2019/05/29 16:55:25][INFO] $ popd
[2019/05/29 16:55:25][INFO] -------------------------------------------------
[2019/05/29 16:55:25][INFO] Running .NET micro benchmarks for 'netcoreapp3.0'
[2019/05/29 16:55:25][INFO] -------------------------------------------------
[2019/05/29 16:55:25][INFO] --------------------------------------------------
[2019/05/29 16:55:25][INFO] Dumping COMPlus environment:
[2019/05/29 16:55:25][INFO]   "COMPlus_TieredCompilation=1"
[2019/05/29 16:55:25][INFO] --------------------------------------------------
[2019/05/29 16:55:25][INFO] $ pushd "/home/asoldatov/performance/src/benchmarks/micro"
[2019/05/29 16:55:25][INFO] $ dotnet run --project /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --configuration Release --framework netcoreapp3.0 --no-restore --no-build -- --filter * --packages /home/asoldatov/performance/artifacts/packages --runtimes netcoreapp3.0
[2019/05/29 16:55:27][INFO] It was not possible to find any compatible framework version
[2019/05/29 16:55:27][INFO] The specified framework 'Microsoft.NETCore.App', version '3.0.0-preview6-27728-04' was not found.
[2019/05/29 16:55:27][INFO]   - No frameworks were found.
[2019/05/29 16:55:27][INFO] 
[2019/05/29 16:55:27][INFO] You can resolve the problem by installing the specified framework and/or SDK.
[2019/05/29 16:55:27][INFO] 
[2019/05/29 16:55:27][INFO] The .NET Core frameworks can be found at:
[2019/05/29 16:55:27][INFO]   - https://aka.ms/dotnet-download
[2019/05/29 16:55:27][ERROR] Process exited with status 150
[2019/05/29 16:55:27][INFO] $ popd
Traceback (most recent call last):
  File "scripts/benchmarks_ci.py", line 278, in <module>
    __main(sys.argv[1:])
  File "scripts/benchmarks_ci.py", line 265, in __main
    args
  File "/home/asoldatov/performance/scripts/micro_benchmarks.py", line 387, in run
    *run_args
  File "/home/asoldatov/performance/scripts/dotnet.py", line 304, in run
    self.working_directory)
  File "/home/asoldatov/performance/scripts/performance/common.py", line 183, in run
    proc.returncode, quoted_cmdline)
subprocess.CalledProcessError: Command '$ dotnet run --project /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --configuration Release --framework netcoreapp3.0 --no-restore --no-build -- --filter * --packages /home/asoldatov/performance/artifacts/packages --runtimes netcoreapp3.0' returned non-zero exit status 150
```

Related to https://github.com/dotnet/coreclr/issues/23520

If I try to use script (or direct call of dotnet) conflicts with installed on system sdk is occurred (even argument `--cli` isn't specified):
```Bash
$ python3 scripts/benchmarks_ci.py -f netcoreapp3.0 --architecture arm --cli /usr/share/dotnet/dotnet  --filter '*'
[2019/05/30 15:31:51][INFO] ----------------------------------------------
[2019/05/30 15:31:51][INFO] Initializing logger 2019-05-30 15:31:51.559127
[2019/05/30 15:31:51][INFO] ----------------------------------------------
[2019/05/30 15:31:51][INFO] Installing tools.
[2019/05/30 15:31:51][INFO] ----------------------
[2019/05/30 15:31:51][INFO] Downloading DotNet Cli
[2019/05/30 15:31:51][INFO] ----------------------
[2019/05/30 15:31:51][INFO] DotNet Install Path: '/home/asoldatov/performance/tools/dotnet/arm'
[2019/05/30 15:31:51][INFO] Downloading https://dot.net/v1/dotnet-install.sh
[2019/05/30 15:31:52][INFO] $ pushd "/home/asoldatov/performance"
[2019/05/30 15:31:52][INFO] $ /home/asoldatov/performance/tools/dotnet/arm/dotnet-install.sh -InstallDir /home/asoldatov/performance/tools/dotnet/arm -Architecture arm -Channel master
[2019/05/30 15:31:54][INFO] dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/3.0.100-preview6-012124/dotnet-sdk-3.0.100-preview6-012124-linux-arm.tar.gz
[2019/05/30 15:31:54][INFO] --2019-05-30 15:31:54--  https://dotnetcli.azureedge.net/dotnet/Sdk/3.0.100-preview6-012124/dotnet-sdk-3.0.100-preview6-012124-linux-arm.tar.gz

[2019/05/30 15:31:55][INFO] Proxy request sent, awaiting response... 200 OK
[2019/05/30 15:31:55][INFO] Length: 126524877 (121M) [application/octet-stream]
[2019/05/30 15:31:55][INFO] Saving to: ‘/tmp/dotnet.DkuhvbpdX’
[2019/05/30 15:31:55][INFO] 
[2019/05/30 15:31:55][INFO]      0K .......... .......... .......... .......... ..........  0%  108K 19m7s
[2019/05/30 15:31:56][INFO]     50K .......... .......... .......... .......... ..........  0% 77.2K 22m53s

									.......... .......... .......... .......... ..........

[2019/05/30 15:45:37][INFO] 123500K .......... .......... .......... .......... .......... 99%  376K 0s
[2019/05/30 15:45:37][INFO] 123550K .........                                             100% 13.2M=13m42s
[2019/05/30 15:45:37][INFO] 
[2019/05/30 15:45:37][INFO] 2019-05-30 15:45:37 (150 KB/s) - ‘/tmp/dotnet.DkuhvbpdX’ saved [126524877/126524877]
[2019/05/30 15:45:37][INFO] 
[2019/05/30 15:45:37][INFO] dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/3.0.100-preview6-012124/dotnet-sdk-3.0.100-preview6-012124-linux-arm.tar.gz
[2019/05/30 15:47:38][INFO] dotnet-install: Adding to current process PATH: `/home/asoldatov/performance/tools/dotnet/arm`. Note: This change will be visible only when sourcing script.
[2019/05/30 15:47:38][INFO] dotnet-install: Installation finished successfully.
[2019/05/30 15:47:38][INFO] $ popd
[2019/05/30 15:47:38][INFO] -----------------------------
[2019/05/30 15:47:38][INFO] Downloading BenchView scripts
[2019/05/30 15:47:38][INFO] -----------------------------
[2019/05/30 15:47:41][INFO] http://benchviewtestfeed.azurewebsites.net/api/v2/package/microsoft.benchview.jsonformat/0.1.0-pre030 -> /home/asoldatov/performance/tools/Microsoft.BenchView.JSONFormat
[2019/05/30 15:47:50][INFO] $ dotnet --info
[2019/05/30 15:47:51][INFO] .NET Core SDK (reflecting any global.json):
[2019/05/30 15:47:51][INFO]  Version:   3.0.100-preview6-012124
[2019/05/30 15:47:51][INFO]  Commit:    fe369022e7
[2019/05/30 15:47:51][INFO] 
[2019/05/30 15:47:51][INFO] Runtime Environment:
[2019/05/30 15:47:51][INFO]  OS Name:     ubuntu
[2019/05/30 15:47:51][INFO]  OS Version:  16.04
[2019/05/30 15:47:51][INFO]  OS Platform: Linux
[2019/05/30 15:47:51][INFO]  RID:         ubuntu.16.04-arm
[2019/05/30 15:47:51][INFO]  Base Path:   /home/asoldatov/performance/tools/dotnet/arm/sdk/3.0.100-preview6-012124/
[2019/05/30 15:47:51][INFO] 
[2019/05/30 15:47:51][INFO] Host (useful for support):
[2019/05/30 15:47:51][INFO]   Version: 3.0.0-preview6-27729-07
[2019/05/30 15:47:51][INFO]   Commit:  7982806713
[2019/05/30 15:47:51][INFO] 
[2019/05/30 15:47:51][INFO] .NET Core SDKs installed:
[2019/05/30 15:47:51][INFO]   3.0.100-preview6-012124 [/home/asoldatov/performance/tools/dotnet/arm/sdk]
[2019/05/30 15:47:51][INFO] 
[2019/05/30 15:47:51][INFO] .NET Core runtimes installed:
[2019/05/30 15:47:51][INFO]   Microsoft.AspNetCore.App 3.0.0-preview6-19279-01 [/home/asoldatov/performance/tools/dotnet/arm/shared/Microsoft.AspNetCore.App]
[2019/05/30 15:47:51][INFO]   Microsoft.NETCore.App 3.0.0-preview6-27729-07 [/home/asoldatov/performance/tools/dotnet/arm/shared/Microsoft.NETCore.App]
[2019/05/30 15:47:51][INFO] 
[2019/05/30 15:47:51][INFO] To install additional .NET Core runtimes or SDKs:
[2019/05/30 15:47:51][INFO]   https://aka.ms/dotnet-download
[2019/05/30 15:47:51][INFO] -------------------------------
[2019/05/30 15:47:51][INFO] Restoring .NET micro benchmarks
[2019/05/30 15:47:51][INFO] -------------------------------
[2019/05/30 15:47:51][INFO] $ pushd "/home/asoldatov/performance/src/benchmarks/micro"
[2019/05/30 15:47:51][INFO] $ dotnet restore /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --packages /home/asoldatov/performance/artifacts/packages
[2019/05/30 15:49:36][INFO]   Restore completed in 1.66 min for /home/asoldatov/performance/src/tools/Reporting/Reporting/Reporting.csproj.
[2019/05/30 15:56:54][INFO]   Restore completed in 8.95 min for /home/asoldatov/performance/src/harness/BenchmarkDotNet.Extensions/BenchmarkDotNet.Extensions.csproj.
[2019/05/30 15:56:57][INFO]   Restore completed in 9.02 min for /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj.
[2019/05/30 15:56:58][INFO] $ popd
[2019/05/30 15:56:58][INFO] --------------------------------------------------
[2019/05/30 15:56:58][INFO] Building .NET micro benchmarks for 'netcoreapp3.0'
[2019/05/30 15:56:58][INFO] --------------------------------------------------
[2019/05/30 15:56:58][INFO] $ pushd "/home/asoldatov/performance/src/benchmarks/micro"
[2019/05/30 15:56:58][INFO] $ dotnet build /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --configuration Release --framework netcoreapp3.0 --no-restore
[2019/05/30 15:57:03][INFO] Microsoft (R) Build Engine version 16.2.0-preview-19278-01+d635043bd for .NET Core
[2019/05/30 15:57:03][INFO] Copyright (C) Microsoft Corporation. All rights reserved.
[2019/05/30 15:57:03][INFO] 
[2019/05/30 15:57:15][INFO]   Reporting -> /home/asoldatov/performance/artifacts/bin/Reporting/Release/netcoreapp3.0/Reporting.dll
[2019/05/30 15:57:21][INFO]   BenchmarkDotNet.Extensions -> /home/asoldatov/performance/artifacts/bin/BenchmarkDotNet.Extensions/Release/netcoreapp3.0/BenchmarkDotNet.Extensions.dll
[2019/05/30 15:57:41][INFO]   MicroBenchmarks -> /home/asoldatov/performance/artifacts/bin/MicroBenchmarks/Release/netcoreapp3.0/MicroBenchmarks.dll
[2019/05/30 15:57:41][INFO] 
[2019/05/30 15:57:41][INFO] Build succeeded.
[2019/05/30 15:57:41][INFO]     0 Warning(s)
[2019/05/30 15:57:41][INFO]     0 Error(s)
[2019/05/30 15:57:41][INFO] 
[2019/05/30 15:57:41][INFO] Time Elapsed 00:00:38.39
[2019/05/30 15:57:41][INFO] $ popd
[2019/05/30 15:57:41][INFO] -------------------------------------------------
[2019/05/30 15:57:41][INFO] Running .NET micro benchmarks for 'netcoreapp3.0'
[2019/05/30 15:57:41][INFO] -------------------------------------------------
[2019/05/30 15:57:41][INFO] --------------------------------------------------
[2019/05/30 15:57:41][INFO] Dumping COMPlus environment:
[2019/05/30 15:57:41][INFO]   "COMPlus_TieredCompilation=1"
[2019/05/30 15:57:41][INFO] --------------------------------------------------
[2019/05/30 15:57:41][INFO] $ pushd "/home/asoldatov/performance/src/benchmarks/micro"
[2019/05/30 15:57:41][INFO] $ dotnet run --project /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --configuration Release --framework netcoreapp3.0 --no-restore --no-build -- --cli /usr/share/dotnet/dotnet --filter * --packages /home/asoldatov/performance/artifacts/packages --runtimes netcoreapp3.0
[2019/05/30 15:57:44][INFO] It was not possible to find any compatible framework version
[2019/05/30 15:57:44][INFO] The specified framework 'Microsoft.NETCore.App', version '3.0.0-preview6-27729-07' was not found.
[2019/05/30 15:57:44][INFO]   - The following frameworks were found:
[2019/05/30 15:57:44][INFO]       3.0.0-preview5-27709-08 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
[2019/05/30 15:57:44][INFO] 
[2019/05/30 15:57:44][INFO] You can resolve the problem by installing the specified framework and/or SDK.
[2019/05/30 15:57:44][INFO] 
[2019/05/30 15:57:44][INFO] The .NET Core frameworks can be found at:
[2019/05/30 15:57:44][INFO]   - https://aka.ms/dotnet-download
[2019/05/30 15:57:44][ERROR] Process exited with status 150
[2019/05/30 15:57:44][INFO] $ popd
Traceback (most recent call last):
  File "scripts/benchmarks_ci.py", line 278, in <module>
    __main(sys.argv[1:])
  File "scripts/benchmarks_ci.py", line 265, in __main
    args
  File "/home/asoldatov/performance/scripts/micro_benchmarks.py", line 387, in run
    *run_args
  File "/home/asoldatov/performance/scripts/dotnet.py", line 304, in run
    self.working_directory)
  File "/home/asoldatov/performance/scripts/performance/common.py", line 183, in run
    proc.returncode, quoted_cmdline)
subprocess.CalledProcessError: Command '$ dotnet run --project /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --configuration Release --framework netcoreapp3.0 --no-restore --no-build -- --cli /usr/share/dotnet/dotnet --filter * --packages /home/asoldatov/performance/artifacts/packages --runtimes netcoreapp3.0' returned non-zero exit status 150
```


If try to remove SDK from host library `libhostfxr.so` cannot be found:
```Bash
[2019/05/30 16:27:44][INFO] Microsoft (R) Build Engine version 16.2.0-preview-19278-01+d635043bd for .NET Core
[2019/05/30 16:27:44][INFO] Copyright (C) Microsoft Corporation. All rights reserved.
[2019/05/30 16:27:44][INFO] 
[2019/05/30 16:27:50][INFO]   Reporting -> /home/asoldatov/performance/artifacts/bin/Reporting/Release/netcoreapp3.0/Reporting.dll
[2019/05/30 16:27:51][INFO]   BenchmarkDotNet.Extensions -> /home/asoldatov/performance/artifacts/bin/BenchmarkDotNet.Extensions/Release/netcoreapp3.0/BenchmarkDotNet.Extensions.dll
[2019/05/30 16:27:52][INFO]   MicroBenchmarks -> /home/asoldatov/performance/artifacts/bin/MicroBenchmarks/Release/netcoreapp3.0/MicroBenchmarks.dll
[2019/05/30 16:27:52][INFO] 
[2019/05/30 16:27:52][INFO] Build succeeded.
[2019/05/30 16:27:52][INFO]     0 Warning(s)
[2019/05/30 16:27:52][INFO]     0 Error(s)
[2019/05/30 16:27:52][INFO] 
[2019/05/30 16:27:52][INFO] Time Elapsed 00:00:07.39
[2019/05/30 16:27:52][INFO] $ popd
[2019/05/30 16:27:52][INFO] -------------------------------------------------
[2019/05/30 16:27:52][INFO] Running .NET micro benchmarks for 'netcoreapp3.0'
[2019/05/30 16:27:52][INFO] -------------------------------------------------
[2019/05/30 16:27:52][INFO] --------------------------------------------------
[2019/05/30 16:27:52][INFO] Dumping COMPlus environment:
[2019/05/30 16:27:52][INFO]   "COMPlus_TieredCompilation=1"
[2019/05/30 16:27:52][INFO] --------------------------------------------------
[2019/05/30 16:27:52][INFO] $ pushd "/home/asoldatov/performance/src/benchmarks/micro"
[2019/05/30 16:27:52][INFO] $ dotnet run --project /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --configuration Release --framework netcoreapp3.0 --no-restore --no-build -- --filter * --packages /home/asoldatov/performance/artifacts/packages --runtimes netcoreapp3.0
[2019/05/30 16:27:55][INFO] A fatal error occurred. The required library libhostfxr.so could not be found.
[2019/05/30 16:27:55][INFO] If this is a self-contained application, that library should exist in [/home/asoldatov/performance/artifacts/bin/MicroBenchmarks/Release/netcoreapp3.0/].
[2019/05/30 16:27:55][INFO] If this is a framework-dependent application, install the runtime in the global location [/usr/share/dotnet] or use the DOTNET_ROOT environment variable to specify the runtime location or register the runtime location in [/etc/dotnet/install_location].
[2019/05/30 16:27:55][ERROR] Process exited with status 131
[2019/05/30 16:27:55][INFO] $ popd
Traceback (most recent call last):
  File "scripts/benchmarks_ci.py", line 278, in <module>
    __main(sys.argv[1:])
  File "scripts/benchmarks_ci.py", line 265, in __main
    args
  File "/home/asoldatov/performance/scripts/micro_benchmarks.py", line 387, in run
    *run_args
  File "/home/asoldatov/performance/scripts/dotnet.py", line 304, in run
    self.working_directory)
  File "/home/asoldatov/performance/scripts/performance/common.py", line 183, in run
    proc.returncode, quoted_cmdline)
subprocess.CalledProcessError: Command '$ dotnet run --project /home/asoldatov/performance/src/benchmarks/micro/MicroBenchmarks.csproj --configuration Release --framework netcoreapp3.0 --no-restore --no-build -- --filter * --packages /home/asoldatov/performance/artifacts/packages --runtimes netcoreapp3.0' returned non-zero exit status 131
```

But if we put this library to appropriate folder, original issue returns:
```Bash
$ /home/asoldatov/performance/tools/dotnet/arm/dotnet run -f netcoreapp3.0 -c Release -- --filter '*'
A fatal error occurred. The required library libhostfxr.so could not be found.
If this is a self-contained application, that library should exist in [/home/asoldatov/performance/artifacts/bin/MicroBenchmarks/Release/netcoreapp3.0/].
If this is a framework-dependent application, install the runtime in the global location [/usr/share/dotnet] or use the DOTNET_ROOT environment variable to specify the runtime location or register the runtime location in [/etc/dotnet/install_location].


$ /home/asoldatov/performance/tools/dotnet/arm/dotnet run -f netcoreapp3.0 -c Release -- --filter '*'
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '3.0.0-preview6-27729-07' was not found.
  - No frameworks were found.

You can resolve the problem by installing the specified framework and/or SDK.

The .NET Core frameworks can be found at:
  - https://aka.ms/dotnet-download
```

--- a/scripts/dotnet.py
+++ b/scripts/dotnet.py
@@ -548,6 +548,7 @@ def install(
     environ['DOTNET_CLI_TELEMETRY_OPTOUT'] = '1'
     environ['DOTNET_MULTILEVEL_LOOKUP'] = '0'
     environ['UseSharedCompilation'] = 'false'
+    environ['DOTNET_ROOT'] = install_dir
 
     # Add installed dotnet cli to PATH
     environ["PATH"] = install_dir + pathsep + environ["PATH"]

